pkg/client/unversioned/events.go

```diff
@@ -184,6 +185,30 @@ func (e *events) Delete(name string) error {
 		Error()
 }
 
+// DeleteCollections deletes a collection of objects.
+func (e *events) DeleteCollection(options *api.DeleteOptions, listOptions unversioned.ListOptions) error {
```

Comment:
Actually nothing. But I don't think it's a problem - if the second one will try to delete object that is already deleted, it will return an error.